tests/conftest.py

```diff
@@ -473,3 +474,21 @@ def toy_dataset_with_mean_shift_in_target():
         "target_0.01": np.concatenate((np.array((-1, 3, 3, -4, -1)), np.array((-2, 3, -4, 5, -2)))).astype(np.float64),
     }
     return TSDataset.to_dataset(pd.DataFrame(df))
+
+
+@pytest.fixture
+def long_hierarchical_structure():
```

Comment:
As you use this fixtures only in test_dataset folder better to create the separate `conftest.py` there, here we store the global fixtures. May be later, when we come to test hierarchical pipeline we will move them here